test/qa_reflection.cpp

```diff
@@ -0,0 +1,55 @@
+/*-*-c++-*-*/
+/*
+ * Copyright 2023 John Sallay
+ *
+ * SPDX-License-Identifier: LGPL-3.0
+ *
+ */
+#include <gtest/gtest.h>
+
+#include <pmtv/pmt.hpp>
+
+#include <fmt/core.h>
+#include <string>
+
+using namespace pmtv;
+
+struct qqq {
+    int x;
+    float y;
+    double z;
+};
+
+REFL_AUTO(type(qqq), field(x), field(y), field(z));
+
+class AA {
```

Comment:
I originally had an additional, more efficient way of doing the reflection.  In talking with Ralph, we decided that it might be too complicated so I pulled it out.  These structs were used there.  I've removed them.